cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/EquipmentExport.java

```diff
@@ -271,7 +271,7 @@ private static void writeShuntCompensators(Network network, String cimNamespace,
                 bPerSection = ((ShuntCompensatorLinearModel) s.getModel()).getBPerSection();
                 gPerSection = ((ShuntCompensatorLinearModel) s.getModel()).getGPerSection();
             }
-            ShuntCompensatorEq.write(s.getId(), s.getNameOrId(), s.getSectionCount(), s.getMaximumSectionCount(), s.getTerminal().getVoltageLevel().getNominalV(), s.getModelType(), bPerSection, gPerSection, cimNamespace, writer);
+            ShuntCompensatorEq.write(s.getId(), s.getNameOrId(), s.getSectionCount().orElseThrow(AssertionError::new), s.getMaximumSectionCount(), s.getTerminal().getVoltageLevel().getNominalV(), s.getModelType(), bPerSection, gPerSection, cimNamespace, writer);
```

Comment:
It is a bit strange that for the export of EQ profile for shunt compensator, we depend on the section count. Same remark for tap changers and tap position. This PR indeed is done for an EQ export without steady state hypothesis.

Reply:
For now, I consider SCADA network not supported everywhere. We will make changes later